src/clients/dotnet/docs.zig

```diff
@@ -0,0 +1,304 @@
+const builtin = @import("builtin");
+const std = @import("std");
+
+const Docs = @import("../docs_types.zig").Docs;
+const run = @import("../shutil.zig").run;
+const run_shell = @import("../shutil.zig").run_shell;
+const script_filename = @import("../shutil.zig").script_filename;
+const write_shell_newlines_into_single_line = @import("../shutil.zig").write_shell_newlines_into_single_line;
+
+pub const DotnetDocs = Docs{
+    .directory = "dotnet",
+
+    .markdown_name = "cs",
+    .extension = "cs",
+    .proper_name = ".NET",
+
+    .test_source_path = "",
+
+    .name = "tigerbeetle-dotnet",
+    .description = 
+    \\The TigerBeetle client for .NET.
+    \\
+    \\Note: The .NET client does not currently support M1/M2 Macs.
```

Comment:
Fixed and tested on a Mac M2 Ventura, we can remove that line now!

Reply:
Done!